openshift/smoke-test.yaml

```diff
@@ -67,27 +89,21 @@ objects:
     namespace: ${NAMESPACE}
     annotations:
       service.alpha.openshift.io/serving-cert-secret-name: web-console-smoke-test-serving-cert
-      prometheus.io/scrape: "true"
-      prometheus.io/scheme: https
   spec:
     ports:
     - name: web
       protocol: TCP
       port: 443
-      targetPort: 3000
+      targetPort: https
     selector:
       app: web-console-smoke-test
     type: ClusterIP
     sessionAffinity: None
-  status:
-    loadBalancer: {}
 parameters:
 - name: IMAGE
-# TODO: update this URL when we migrate up the repo
-  value: benjaminapetersen/origin-web-console-smoke-test:latest
+  required: true
```

Comment:
do we want to defaul this to the `quay.io/redhat/origin-web-console-smoke-test:latest` (that we have not yet pushed)?